File: app/src/main/java/com/sendbird/calls/quickstart/main/SettingsFragment.java
package com.sendbird.calls.quickstart.main;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.sendbird.calls.SendBirdCall;
import com.sendbird.calls.User;
import com.sendbird.calls.quickstart.R;
import com.sendbird.calls.quickstart.utils.ActivityUtils;
import com.sendbird.calls.quickstart.utils.AuthenticationUtils;
import com.sendbird.calls.quickstart.utils.UserInfoUtils;

public class SettingsFragment extends Fragment {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_settings, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        User currentUser = SendBirdCall.getCurrentUser();
        if (currentUser != null) {
            UserInfoUtils.setProfileImage(getActivity(), currentUser, view.findViewById(R.id.image_view_profile));
            ((TextView)view.findViewById(R.id.text_view_user_id)).setText(currentUser.getUserId());
            ((TextView)view.findViewById(R.id.text_view_nickname)).setText(currentUser.getNickname());
        }

        view.findViewById(R.id.linear_layout_application_information).setOnClickListener(view1 -> {
            if (getActivity() != null) {
                ActivityUtils.startApplicationInformationActivity(getActivity());
            }
        });

        view.findViewById(R.id.linear_layout_sign_out).setOnClickListener(view1 -> {
            AuthenticationUtils.deauthenticate(getActivity(), isSuccess -> {
                if (getActivity() != null) {
                    ActivityUtils.startAuthenticateActivityAndFinish(getActivity());
                }
            });
        });
    }
}

File: directcall-kotlin/app/src/main/java/com/sendbird/calls/quickstart/main/SettingsFragment.kt
package com.sendbird.calls.quickstart.main

import android.os.Bundle
import android.view.View
import com.sendbird.calls.SendBirdCall.currentUser
import com.sendbird.calls.quickstart.BaseFragment
import com.sendbird.calls.quickstart.databinding.FragmentSettingsBinding
import com.sendbird.calls.quickstart.utils.AuthenticationUtils.deauthenticate
import com.sendbird.calls.quickstart.utils.setNickname
import com.sendbird.calls.quickstart.utils.setProfileImage
import com.sendbird.calls.quickstart.utils.startApplicationInformationActivity
import com.sendbird.calls.quickstart.utils.startAuthenticateActivityAndFinish

class SettingsFragment : BaseFragment<FragmentSettingsBinding>(FragmentSettingsBinding::inflate) {

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        val currentUser = currentUser
        if (currentUser != null) {
            setProfileImage(currentUser, binding.imageViewProfile)
            setNickname(currentUser, binding.textViewNickname)
            binding.textViewUserId.text = currentUser.userId
        }
        binding.linearLayoutApplicationInformation.setOnClickListener {
            startApplicationInformationActivity()
        }
        binding.linearLayoutSignOut.setOnClickListener {
            deauthenticate(requireContext()) {
                startAuthenticateActivityAndFinish()
            }
        }
    }
}